### ISSUE TITLE
Cassandra chart not working while enabling the TLS

### ISSUE THREAD

**Issue by Jenishk56** [bitnami/charts#2423]
**Which chart**:
bitnami cassandra chart with version 3.11.6

**Describe the bug**
While trying Cassandra bitnami in GKE without TLS enabled, everything works fine. However, as suggested after generating the keystone and truststore, recreating facing and error with `opt/bitnami/scripts/libcassandra.sh` I believe. It throws an error with sed command.

The error is as below
Cassandra 17:01:51.79 INFO  ==> Initializing Cassandra database...
sed: -e expression #1, char 89: unknown option to `s'```

Below values have been set in the chart 
```
tlsEncryptionSecretName: cassandra-tls
clientEncryption: true
```
I believe the "sed -E " in libcassandra.sh is creating a problem somehow. Is that something that could be set from the chart itself to resolve this error?

**Comment by marcosbc** [comment]
Hi @Jenishk56, could you check if the contents of the cassandra-tls secret include an '@' symbol?

Currently we are running a command like the following, which could explain the issue if that happens to be the case:

```
sed -E "s@^(#\s)?(\s*)(\-\s*)?truststore_password:.*@\2\3truststore_password: 'CONTENTS OF CASSANDRA_TRUSTSTORE_PASSWORD'@"
```

**Comment by Jenishk56** [comment]
Hi @marcosbc 

Keystore and truststore I used, are generated using keytools with ".jks" format. Those are in binaries and aren't human-readable.

Is there a way to generate the plain text "keystone" and "truststore" with a cert file that can be used as a Cassandra client?

Thanks in advanced,
Jenish

**Comment by Jenishk56** [comment]
@marcosbc  Thank you for getting back.

I think, that might be the issue :)

**Comment by marcosbc** [comment]
Thanks @Jenishk56! I'm going to create an internal task for fixing this issue, but for now as long as you avoid "@" in your passwords, you should be fine.

**Comment by marcosbc** [comment]
We've released Cassandra Chart 5.4.1 which includes a fix for this issue. It includes the container image `bitnami/cassandra:3.11.6-debian-10-r92`.

Basically the main change is using a non-printable ASCII character as a sed delimiter (which should not appear in the regex nor substitution text), so now it is possible to use characters such as `@`, `|` and others.
